frontend/src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -40,7 +40,7 @@ type Props = {
 /**
  * Generic confirmation modal
  * controls its own open state
- * calls onDeny on cancel, esc, or overlay click
```

Comment:
Looks like Radix' `AlertDialog` is intentionally not dismissible from clicking on the overlay. They do have that behaviour on regular `Dialog`s.